kmp-library/src/commonTest/kotlin/com/novoda/aws/v4/signer/hash/Sha256Test.kt

```diff
@@ -18,7 +18,7 @@ class Sha256Test {
     @ExperimentalUnsignedTypes
     @Test
     fun `get sha-256 from request`() {
-        val contentHash = Sha256.digestAsHexString(request)
+        val contentHash = Sha256Encoder.encode(request)
```

Comment:
Should this test file be renamed to `Sha256EncoderTest`?

Reply:
Done in 38d0d6f